File: API/PageAndRDL/ReportWizard/VB.NET/ReportWizard/MetaData/FieldMetaData.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.Text
Namespace MetaData
	Public Class FieldMetaData

		Private _title As String
		Public Sub New()
			Name = ""
			Title = ""
			IsNumeric = False
			AllowTotaling = False
			FormatString = ""
			PreferredWidth = "1cm"
			SummaryFunction = ""
		End Sub

		Public Name As String

		'property for data binding purposes
		Public Property Title() As String
			Get
				Return _title
			End Get
			Set(ByVal value As String)
				_title = value
			End Set
		End Property
		Public IsNumeric As Boolean
		Public AllowTotaling As Boolean
		Public FormatString As String
		Public PreferredWidth As String
		Public SummaryFunction As String
	End Class
End Namespace

File: API/PageAndRDL/ReportWizard/C#/ReportWizard/MetaData/FieldMetaData.cs

namespace GrapeCity.ActiveReports.Samples.ReportWizard.MetaData
{
	public class FieldMetaData
	{
		private string _title;

		public FieldMetaData()
		{
			Name = "";
			Title = "";
			IsNumeric = false;
			AllowTotaling = false;
			FormatString = "";
			PreferredWidth = "1cm";
			SummaryFunction = "";
		}

		public string Name;

		// property for data binding purposes
		public string Title
		{
			get { return _title; }
			set { _title = value; }
		}

		public bool IsNumeric;
		public bool AllowTotaling;
		public string FormatString;
		public string PreferredWidth;
		public string SummaryFunction;
	}
}
